lib/venice/in_app_receipt.rb

```diff
@@ -52,7 +52,8 @@ def initialize(attributes = {})
       @version_external_identifier = attributes['version_external_identifier']
 
       # expires_date is in ms since the Epoch, Time.at expects seconds
-      @expires_at = Time.at(attributes['expires_date'].to_i / 1000) if attributes['expires_date']
+      @expires_at = Time.at(attributes['expires_date_ms'].to_i / 1000) if attributes['expires_date_ms']
```

Comment:
@ohbarye How do you think the above comment? I guess, this API considers to adjust for `Time.at`. In example with https://github.com/nomad/venice/pull/30/files#diff-844fe150cfb2508a2496a02abfccdaeaR23.

```
[1] pry(main)> Time.at(1403941673000)
46459-02-21 07:03:20 +0900
```